### PR TITLE
MESH-1660/External Agents Integration Test

### PR DESCRIPTION
**Pull request by JMoore96**
Creating a integration test in Typescript for the external agents module

### CODE REVIEW

**Comment by adamdossa** [comment]
@JMoore96 to check why CI isn't running the new EA integration tests

**Comment by Neopallium** [comment]
@JMoore96 The integration-tests are using these commands to run:
```
timeout 5m npm install --verbose && timeout 5m npm run build && timeout 30m npm test
```
Easier to view format:
```
npm install --verbose
npm run build
npm test
```
Is that missing any steps to install, build and run the tests?

**Comment by Neopallium** [comment]
@adamdossa It looks like there is still an issue with the integration_test pipeline.  I looked at the logs and test `11_B_settlement` doesn't seem to finish running and `12_external_agents` is not even started.

File: scripts/cli/README.md [hunk 1]
```diff
@@ -42,158 +44,20 @@ The test scripts include those named below, and most make up `npm test`:
...
+ - 9_bridge_transfer
+ - 10_governance_management
+ - 11_A_settlement
+ - 11_B_settlement
+ - 12_external_agents
 
  To run a single script:
  ```shell
-$ npm run -s 0-create-identities
+$ npm run 0_create-_dentities
```
[the remark this answers is not in the document]
Comment (JMoore96) [reply scripts/cli/README.md:56]: you are right, thanks

**Comment by F-OBrien** [comment]
> @adamdossa It looks like there is still an issue with the integration_test pipeline. I looked at the logs and test `11_B_settlement` doesn't seem to finish running and `12_external_agents` is not even started.

Do tests 11_A and 11_B even do what they are supposed to? I don't see any uniqueness claims being added for the tickers and some identities are missing attestations so while settlements will get scheduled successfully the subsequent transfers will fail and balances won't update. Or did I miss something?

**Comment by adamdossa** [comment]
@Neopallium @JMoore96 @F-OBrien This PR now addresses:
- bug in `missing_scope_claim` for non-uniqueness enforcing assets
- fixes to integration tests for settlement to run without IU claims
- fixes for CI pipeline to run to completion

For the last part, I don't really understand the root cause - best guess is that Concourse is killing the process if it doesn't see output for a period of time, but I can't find where that's being configured in our pipelines - work around is to do more logging.

**Review by adamdossa — APPROVED** [review]

**Review by Neopallium — APPROVED** [review]

**Pull request merged**